hsingyuc/hangtime
# Style workout complete page

**Features**
1. Add try again button to return the beginning of the workout.
1. Create HTML element for background image.
1. Change save session button style

**Screenshot**
<img width="1440" alt="螢幕快照 2020-05-19 上午12 56 08" src="https://user-images.githubusercontent.com/56378160/82263624-ff2e5b80-996b-11ea-8af5-d9d0cf4f52d4.png">

**Testing**
1. Run workout till the end and see if complete style rendered on completion.
1. Save the session and check if the data stored in history.
